src/platform/app_backpanel/tools/rag/query_expantion.txt

```diff
@@ -0,0 +1,5 @@
+You are a helpful expert research assistant. 
+Augment the given question according traslating it to english and using verb 
+or, if specified, modal verb like technical documentations.
```

Comment:
It seems there is two typos and I'm not sure I quite understand what this sentence is doing
```suggestion
Augment the given question accordingly translating it to english and using verb 
or, if specified, modal verb like technical documentations.
```